tools/lib/html_branches.py

```diff
@@ -24,8 +24,8 @@ class HtmlTreeBranch(object):
     conceptually be something like "p div(#yo) span(.bar)".
     """
 
-    def __init__(self, tags, fn):
-        # type: (List[TagInfo], str) -> None
+    def __init__(self, tags, fn):  # FIXME: fn is not really used and could be removed?
+        # type: (List[TagInfo], Optional[str]) -> None
```

Comment:
this parameter is used in `/tools/lib/html_grep.py` basically while grepping it is used to print filenames a particular html branch belongs to.
I guess we should leave it there.